.github/scripts/muffet.sh

```diff
@@ -42,5 +42,6 @@ muffet http://localhost:1313 \
   --exclude "https://tools.google.com.*" \
   --exclude "https://fhir.org/" \
   --exclude "https://docs.google.com/spreadsheets/d/12345ABCDEF/.*" \
+  --exclude "https://docs.google.com/document/d/14AuJ7SerLuOPESBjQlJqpBtzwSAoVf5ykTT7fjyJBT0/*" \
   --exclude "https://drive.google.com/file/d/1YPXoba9gVmD7SP-X88PpJIsIVGvY86_G.*" \
-  --exclude "https://doi.org/10.1080/02681102.2019.1667289"
+  --exclude "https://doi.org/10.1080/02681102.2019.1667289" \
```

Comment:
I wasn't expecting to see this escape at the end of the script. Was this intentional?

Reply:
d'oh!  nice catch. fixed in 07a16c3d